randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -180,6 +180,8 @@ Extra - asset_id: collision_camera_003
       Any of the following:
           Flash Shift or Gravity Suit or Spider Magnet or Simple IBJ or Use Spin Boost
           Grapple Beam and Grapple Movement (Beginner)
+          # Damage Boost (Beginner): https://youtu.be/qIanrM7cc3I
+          Damage Boost (Beginner)
```

Comment:
Should probably have a damage requirement on it though.

Reply:
Added!